StortSpelProjekt/Engine/src/ECS/SceneManager.cpp

```diff
@@ -116,7 +116,7 @@ void SceneManager::ChangeScene()
 {
 	if (m_ChangeSceneNextFrame)
 	{
-		if (m_pActiveScene->GetName() == "ShopScene" || m_pActiveScene->GetName() == "GameScene")
+		if (m_ActiveScenes[0]->GetName() != "gameOverScene")
```

Comment:
this should be m_pActiveScene, also gave me a compilation error

Reply:
Fixed